### PR TITLE
fix: "wrong number of arguments" must be in lower case

### PR DESCRIPTION
**Pull request by lsvmello**
Signed-off-by: Leonardo Mello <lsvmello@gmail.com>

This fixes https://github.com/dragonflydb/dragonfly/issues/390.

Since the print pointed out just the "wrong number of arguments" message I lowered the command just for the `WrongNumArgsError` function.

Should the command also be lowered on `InvalidExpireTime` and `UnknownSubCmd`? If that's the case then is it ok to create a `ToLower` function for `string_view` types on the `facade.cc` or is it better to put it on another file?

I couldn't execute the `facade_test` also, it compiles but doesn't create any executable.

First open source PR ever so if there is any improvement just say it and  I'll be glad work on it.

### CODE REVIEW

File: src/facade/facade.cc [hunk 1]
```diff
@@ -51,7 +51,9 @@ ConnectionStats& ConnectionStats::operator+=(const ConnectionStats& o) {
 #undef ADD
 
 string WrongNumArgsError(string_view cmd) {
-  return absl::StrCat("wrong number of arguments for '", cmd, "' command");
+  string lcmd(cmd);
+  absl::AsciiStrToLower(&lcmd);
```
Comment (dranikpg) [inline src/facade/facade.cc:55]: Abseil already has a one-liner for your use case https://github.com/abseil/abseil-cpp/blob/2fc358dab079c220feb81f75b5a366f8d1464cd4/absl/strings/ascii.h#L170-L175

**Review by dranikpg — COMMENTED** [review]
Hi!

1. Its great you pointed out the neighbouring functions as well. If you run `SET k v EX -1` on redis, you'll see that it returns `ERR invalid expire time in 'set' command`. In case with `UnknownSubCmd` the command is returned in original form. So `InvalidExpireTime` can be changed as well
2. See my comment. No need for a separate function
3. There is no real facade_test, its a helper library for other tests

File: src/facade/facade.cc (hunk 1 above)
Comment (lsvmello) [reply src/facade/facade.cc]: Oh I haven't seem that. Fixed!
And also changed lowered the command on the InvalidExpireTime

**Comment by romange** [comment]
@lsvmello Do you mind merging all the commits into a single one on your side? I could do it too but then the commit won't track to you, imho.

**Pull request closed without merging**